power_grid_model_c/power_grid_model_c/include/power_grid_model_c/dataset.h

```diff
@@ -83,6 +83,47 @@ PGM_API PGM_Idx PGM_dataset_info_elements_per_scenario(PGM_Handle* handle, PGM_D
  */
 PGM_API PGM_Idx PGM_dataset_info_total_elements(PGM_Handle* handle, PGM_DatasetInfo const* info, PGM_Idx component_idx);
 
+/**
+ * @brief Return whether a component has attribute indications.
+ *
+ * Attribute indications are used to indicate the presence of meaningful attributes for a certain component in the
+ * dataset. It returns 1 for a component if all the elements of all the scenarios for this component are list-like in
+ * the serialized data.
```

Comment:
```suggestion
 * dataset. If it returns 1 for that component, you can create only the columnar buffers with the indicated attributes. In this case, you are guaranteed that all information will be retained. Otherwise, no such guarantees are made.
```